### ISSUE TITLE
There was an issue fetching that device. Make sure your account is linked and the ID is correct.

### ISSUE THREAD

**Issue by de-salvatierra** [TuyaAPI/cli#129]
```
[ Error: sign invalid
      at OpenAPI._client.got_1.default.extend.hooks.afterResponse.response (/usr/local/lib/node_modules/@tuyapi/cli/node_modules/@tuyapi/openapi/dist/api.js:74:31)
      at EventEmitter.emitter.on (/usr/local/lib/node_modules/@tuyapi/cli/node_modules/got/dist/source/as-promise.js:87:38)
      at process._tickCallback (internal/process/next_tick.js:68:7),
  Error: sign invalid
      at OpenAPI._client.got_1.default.extend.hooks.afterResponse.response (/usr/local/lib/node_modules/@tuyapi/cli/node_modules/@tuyapi/openapi/dist/api.js:74:31)
      at EventEmitter.emitter.on (/usr/local/lib/node_modules/@tuyapi/cli/node_modules/got/dist/source/as-promise.js:87:38)
      at process._tickCallback (internal/process/next_tick.js:68:7),
  Error: sign invalid
      at OpenAPI._client.got_1.default.extend.hooks.afterResponse.response (/usr/local/lib/node_modules/@tuyapi/cli/node_modules/@tuyapi/openapi/dist/api.js:74:31)
      at EventEmitter.emitter.on (/usr/local/lib/node_modules/@tuyapi/cli/node_modules/got/dist/source/as-promise.js:87:38)
      at process._tickCallback (internal/process/next_tick.js:68:7),
  Error: sign invalid
      at OpenAPI._client.got_1.default.extend.hooks.afterResponse.response (/usr/local/lib/node_modules/@tuyapi/cli/node_modules/@tuyapi/openapi/dist/api.js:74:31)
      at EventEmitter.emitter.on (/usr/local/lib/node_modules/@tuyapi/cli/node_modules/got/dist/source/as-promise.js:87:38)
      at process._tickCallback (internal/process/next_tick.js:68:7) ]```
thx

**Comment by de-salvatierra** [comment]
To better secure your cloud development, Tuya will upgrade the signature algorithm. Projects created after June 30, 2021 are subject to signature verification with the new algorithm

**Comment by de-salvatierra** [comment]
https://developer.tuya.com/en/docs/iot/singnature?id=Ka43a5mtx1gsc

**Comment by codetheweb** [comment]
Fixed in 1.17.0.

See: https://github.com/codetheweb/tuyapi/issues/492
